lms/djangoapps/learner_home/test_serializers.py

```diff
@@ -938,13 +924,19 @@ def make_test_context(self, enrollments=None, enrollments_with_entitlements=None
             for entitlement in all_entitlements
         }
 
+        programs = {
```

Comment:
Maybe make a new kwarg to this function, like `has_programs` or something, so we can toggle having programs or not